src/subprocess_launcher.rs

```diff
@@ -0,0 +1,51 @@
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use SubprocessLauncher;
+use ffi;
+use glib;
+use glib::object::IsA;
+use GtkRawFd;
```

Comment:
I don't like this one, and it will also make the API unsafe as you can pass random integers in there now. There's a reason why I said to use the proper traits for this :)